src/Bravo3/Orm/Drivers/PubSubDriverInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace Bravo3\Orm\Drivers;
+
+interface PubSubDriverInterface
+{
+    const SUBSCRIPTION_PATTERN = 'BRAVO3_EVT-*';
+
+    /**
+     * Start listening to subscribed channels of the database (if PubSub is supported)
+     * @return void
+     */
+    public function listenToPubSub(callable $callback);
+
+    /**
+     * Publishes a message to the configured channel. Channel and Message length is limited based on the driver used.
+     *
+     * @param string $channel
+     * @param string $message
+     * @return int
+     */
+    public function publishMessage($channel, $message);
```

Comment:
Explain a bit more? Unit tests should be for the driver, which we assume works as it is from Predis.